packages/session-recorder/package.json

```diff
@@ -34,7 +34,7 @@
 		"type-fest": "^4.37.0"
 	},
 	"peerDependencies": {
-		"@splunk/otel-web": "^0.23.0-alpha.2"
+		"@splunk/otel-web": "workspace:*"
```

Comment:
pnpm will replace it with the current version, but we will be missing the ^. I think that's okay, since we strictly require the exact same version to be installed.